### ISSUE TITLE
ENOENT: no such file or directory

### ISSUE THREAD

**Issue by BenjaminSAILcloud** [Microsoft/vsts-react-native-tasks#22]
Hi,
when I try to use React Native Prepare after a NPM install, the task fail with this error : 
```
ENOENT: no such file or directory, open '/Users/runner/runners/2.164.7/work/1/s/node_modules/react-native/React/React.xcodeproj/project.pbxproj'
```
But the `Xcode Project(s)`parameter is not set to this path, I don't understand why it triggers this error.

v0.1.1

**Comment by joe-knd** [comment]
I'm having the same issue, is there any news about this?

**Comment by F0ofoon** [comment]
@joganet  No, and the project is discontinued anyway so I didn't use it at all. 
<img width="627" alt="image" src="https://user-images.githubusercontent.com/44926723/82788773-661aab80-9e69-11ea-8008-98bfc233e0ac.png">
Here is my pipeline and it works perfectly for me.
